test/WebJobs.Script.Tests/TestScripts/CSharp/AssembliesFromSharedLocation/function.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bindings": [
```

Comment:
To prevent an explosion of functions for various assembly load permutations, wondering it you can't just add this shared assembly scenario to an existing test function? Kind of like what I'm doing with the Node "Scenarios" function (https://github.com/Azure/azure-webjobs-sdk-script/tree/dev/test/WebJobs.Script.Tests/TestScripts/Node/Scenarios)